hs/src/Reach/SimClient.hs

```diff
@@ -0,0 +1,37 @@
+{-# OPTIONS_GHC -Wno-missing-export-lists #-}
```

Comment:
You should write this program in JS

Reply:
where in the project  filespace does it go? @jeapostrophe